docs/using.rst

```diff
@@ -55,7 +55,8 @@ standalone_ Y    N    | Uses a "standalone" webserver to obtain a cert. Requires
                       | with no webserver, or when direct integration with the local
                       | webserver is not supported or not desired.
 manual_     Y    N    | Helps you obtain a cert by giving you instructions to perform http-01_ (80) or
-                      | domain validation yourself.                                   dns-01_ (53)
+                      | domain validation yourself. Additionally allows you to        dns-01_ (53)
+                      | specify scripts to be run before and after validation.
```

Comment:
Not just "scripts to be run before and after validation", but "scripts to let you automate the validation task in a customized way", or something to that effect?